napari/_vispy/visuals/surface.py

```diff
@@ -0,0 +1,8 @@
+from .clipping_planes_mixin import ClippingPlanesMixin
+from .mesh import Mesh
+
+
+class SurfaceVisual(ClippingPlanesMixin, Mesh):
```

Comment:
Could `VectorsVisual` also be defined this way, or is there a reason that a vectors layer could/should not be clipped?

Reply:
Oops. This is actually the visual that should be used by the vectors layer :)